configuration/config.yml

```diff
@@ -8,6 +8,7 @@ server:
       - type: access-logstash-console
 serviceInfo:
   name: config
+userHubSessionDuration: 90m
```

Comment:
Surely this is in config somewhere already?

Reply:
Nope 🤷‍♂ 
It's already hardcoded in two other config yamls - making this a global config value is not in scope of this PR (I would do it anyway but this has to go out _today_ so not the best time)

Reply:
Understood - if you've found where those places are it might be worth a TODO somewhere to tie them all together.

Reply:
I might push another PR after all this is done - as we'll need a new endpoint on the config service